photutils/psf/sandbox.py

```diff
@@ -304,7 +304,7 @@ class Reproject(object):
 
     Parameters
     ----------
-    wcs_original, wcs_rectified : `~astropy.wcs.WCS` or `~gwcs.wcs.WCS`
+    wcs_original, wcs_rectified : `~astropy.wcs.WCS` or ``gwcs.wcs.WCS``
```

Comment:
I don't think this class is exposed in the docs, but that's a better long-term solution.  Done, thanks.